var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -160,18 +160,3 @@ def cache_test_sources(self):
         """Copy the example matrices after the package is installed to an
```

Comment:
Also, this method is would no longer be needed if you remove the stand-alone tests since it's sole purpose is to add the files that are used by it.